src/widgets/phpinfo/PhpInfoDataNormalizer.php

```
@@ -330,6 +360,156 @@ private static function buildTiles(array $rawTiles, string $home): array
+            self::TABLE_KIND_DATA => $count === 1 ? 'row' : 'rows',
+            self::TABLE_KIND_NOTE => $count === 1 ? 'note' : 'notes',
+            default => $count === 1 ? 'value' : 'values',
+        };
+
+        return "$count $noun";
+    }
+
+    private static function isSensitiveVariableKey(string $key): bool
+    {
+        $isRuntimeVariable = str_starts_with($key, '$_');
+        $isEnvironmentVariable = $key !== '' && strtolower($key) !== $key;
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $isEnvironmentVariable. Keep variable name length under 20. (undefined)

(LongVariable)